test/pkg/environment/common/setup.go

```diff
@@ -65,6 +66,8 @@ var (
 		&schedulingv1.PriorityClass{},
 		&corev1.Node{},
 		&v1.NodeClaim{},
+		&admissionregistrationv1.ValidatingAdmissionPolicy{},
```

Comment:
We have a validatingAdmissionPolicy now?

Reply:
Yeah, this is how we will mocking node registration failure